python/lsst/ts/externalscripts/characterization/scripts.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
```

Comment:
Please, rename this file to reflect what is inside it. I suggest `laser_characterization.py`